colossus-metrics/src/main/scala/colossus/metrics/MetricSystem.scala

```diff
@@ -218,14 +218,30 @@ object MetricSystem {
 }
 
 //has to be a better way
```

Comment:
This object grew a bit, as `ServerSetting` had to expand this a bit. Having this here feels weird, but there is no other location where this can be shared between metrics and core.